pyproject.toml

```diff
@@ -1,35 +1,38 @@
-[tool.poetry]
+[project]
 name = "blurring_as_a_service"
-version = "2.0.0"
+version = "2.1.0"
 description = ""
 authors = [
-    "Sebastian Davrieux <s.davrieux@amsterdam.nl>",
-    "Andrea Lombardo <a.lombardo@amsterdam.nl>",
-    "Diana Epureanu <d.epureanu@amsterdam.nl>",
-    "Chris Eijgenstein",
-    "Daan Bloembergen <d.bloembergen@amsterdam.nl>"
+    {name = "Sebastian Davrieux", email = "s.davrieux@amsterdam.nl"},
+    {name = "Diana Epureanu", email = "d.epureanu@amsterdam.nl"},
+    {name = "Andrea Lombardo", email = "a.lombardo@amsterdam.nl"},
+    {name = "Daan Bloembergen", email = "d.bloembergen@amsterdam.nl"},
+    {name = "Chris Eijgenstein", email = ""},
+]
+maintainers = [
+    { name = "Computer Vision Team, Gemeente Amsterdam", email = "cvt@amsterdam.nl" },
+]
+requires-python = ">=3.8, <=3.12"
+dependencies = [
+    "aml_interface",
+    "cvtoolkit",
+    "yolo_model_development_kit",
+    "azure-cli>=2.67.0",
+    "psycopg2>=2.9.10",
+    "azureml-core>=1.59.0",
 ]
 
-[tool.poetry.dependencies]
-python = "~3.12"
-aml_interface = { git = "https://github.com/Computer-Vision-Team-Amsterdam/AML-Interface.git", tag = "v1.1.2" }
-cvtoolkit = { git = "https://github.com/Computer-Vision-Team-Amsterdam/CVToolkit.git",  tag = "v2.0.4" }
-yolo_model_development_kit = { git = "https://github.com/Computer-Vision-Team-Amsterdam/YOLO-model-development-kit.git", tag = "v1.1.2" }
-azure-cli = "^2.67.0"
-psycopg2 = "^2.9.10"
-azureml-core = "^1.59.0"
+[tool.uv.sources]
+torch = { index = "pytorch" }
```

Comment:
I think this is not needed with the new base image in Dockerfile, now we can use the default torch from pypi.